tests/.suitespec.json

```diff
@@ -38,8 +38,6 @@
             "ddtrace/settings/config.py",
             "ddtrace/settings/http.py",
             "ddtrace/settings/integration.py",
```

Comment:
Ah! It was this comma! Perhaps we should add a JSON validation step 😢

Reply:
argh!